invenio_previewer/__init__.py

```diff
@@ -169,143 +181,141 @@
+points and define its priority.
+
+Let's try to create a ``.txt.`` file previewer. We need to implement:
 
-1. ``can_preview()`` is called in order to check if a given file can be
+1. ``can_preview()`` method: called in order to check if a given file can be
    previewed and should return a boolean.
-2. ``preview()`` is called to actually render the preview.
-3. ``previewable_extensions`` is a list of string saying which files extensions
-   can preview.
-
-Both methods is passed a ``PreviewFile`` instance, which contains the extract
-file dictionary, the record and the persistent identifier. ``PreviewFile`` also
-provides a method which can be used to open the file, in case it is managed by
-Invenio-Files-REST.
-
-For our ``txt`` previewer, we can create a file with the following content:
-
->>> previewable_extensions = ['txt']
->>> def can_preview(file):
-...     return file.file['uri'].endswith('.txt')
->>> def preview(file):
-...     fp = file.open()
-...     content = fp.read().decode('utf-8')
-...     fp.close()
-...     return content
-
-Configuration:
-Once we have our code, we should register our previewer in the previwer entry
-point. So, go to your ``setup.py`` and create a new entry (If it doesn't exist)
-to declare ``invenio_previewer.previewers`` entry points. Then, you should add
-a new entry with specifying the python path of your module:
->>> 'invenio_previewer.previewers': [
->>>     'tex_previewer = myproject.modules.previewer.extensions.txt_previewer',
->>> ]
-
-The configuration above made it is only making to our project to be aware of
-the module but the previewer can not be used. As we said before, you need to
-add it to ``PREVIEWER_PREFERENCES`` in the correct position. The first position
-is going to be perfect in the case of this TXT previewer:
->>> PREVIEWER_PREVIEWERS_ORDER=
->>>     [
->>>         'invenio_previewer.extensions.csv_dthreejs',
->>>         'invenio_previewer.extensions.json_prismjs',
->>>         'invenio_previewer.extensions.xml_prismjs',
->>>         'invenio_previewer.extensions.simple_image',
->>>         'invenio_previewer.extensions.mistune',
->>>         'invenio_previewer.extensions.pdfjs',
->>>         'invenio_previewer.extensions.zip',
->>>         'invenio_previewer.extensions.default',
->>>     ]
-
-Now, the previewer is ready to be used.
-
-Bundles:
-In the previously described previewer, we were returning a simple string as
-response  but a real implementation should return a HTML. This HTML should
-extends ``invenio_previewer/abstract_previewer.html`` in order to take
-advantage of many presentation features.
-
-But when you are defining a new template, maybe you are requiring some files
-like javascript or style documents. For those cases, you need to create a
-bundle. Check ``Invenio-Assets`` out to learn how to add them.
+2. ``preview()`` method: called to render the preview.
+3. ``previewable_extensions`` property: list of strings to define which files
```

Comment:
again I think singular form of `files` would be more fitting, since it's the extensions that are plural.